packages/glob/src/internal-path.ts

```diff
@@ -1,6 +1,6 @@
-import * as assert from 'assert'
 import * as path from 'path'
 import * as pathHelper from './internal-path-helper'
+import assert from 'assert'
```

Comment:
reaction due to moving `"esModuleInterop": true` to the root `tsconfig.json` - which previously was only defined in `packages/github/tsconfig.json`